transfers/file-transfers.md

```diff
@@ -99,19 +110,35 @@ Or
 
 * /ACQ_1234_123456
   * /metadata
-  * /objects
+  * /objects-->
+
```

Comment:
Maybe delete all the commented out text. The .md is bulky and difficult to navigate.